LightPanel-21.dsa

```diff
@@ -1801,7 +1801,7 @@ else {
 	
 	// Create the cancel push button
 	var wCancelBtn = new DzPushButton( wDlgBtnsGB );
-	wCancelBtn.text = "&Cancel";
+	wCancelBtn.text = "Cancel";
```

Comment:
The Ampersand used to mean it also coudl be used as a keyboard accelerator. Has this been disabled in newer Daz ?
Did you need to do it to get it to work ?

Reply:
E.g. Alt-A or ctrl-shift-A